### DISCUSSION TITLE
Does mimir have metric of total series/samples in his object store?

### DISCUSSION THREAD

**Discussion by MalKeshar** [grafana/mimir#4929] [Help and support] [answered]
Does mimir have metric of total series/samples in object store?

In prometheus we have prometheus_tsdb_head_series_created_total.

cortex_ingester_active_series and cortex_ingester_memory_series_created_total not the same.

This information is available for example at mimir API /store-gateway/tenant/$tenant_id/blocks on per tenant basis, but I fail to find appropriate metric in source code for storegateway component

**Comment by pracucci** [comment, marked as answer]
Mimir currently doesn't expose a metric with the total series/samples in the object storage. On the other hand, it expose metrics about the in-memory series in ingesters (`cortex_ingester_active_series` and `cortex_ingester_memory_series_created_total - cortex_ingester_memory_series_removed_total`). Also the compactor expose metrics about the total number of blocks in the storage (`max by(user) (cortex_bucket_blocks_count)`).

**Reply by MalKeshar** [reply]
Thank you for answer
